docs/nuget-org/package-readme-on-nuget-org.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Package Readme on NuGet.org
+description: Detailed explanation of how Readme files on NuGet.org are rendered and what to do when you run into issues.
+author: chgill-MSFT
+ms.author: chgill
+ms.date: 02/23/2021
+ms.topic: conceptual
+ms.reviewer: anangaur
+---
+
+# Package Readme on NuGet.org
+
+[Include a Readme file in your NuGet package](https://docs.microsoft.com/nuget/reference/msbuild-targets#packagereadmefile) to make your package details richer and more informative for your users!
+
+This is likely one of the first elements users will see when they view your package details page on NuGet.org and is essential to making a good impression!
+
+## What should my Readme include?
+
+Consider including the following items in your readmes:
+* An introduction to what your package is and does - what problems does it solve?
+* How to get started with your package - are there any specific requirements?
+* Links to more comprehensive documentation if not included in the readme itself.
+* At least a few code snippets/samples or example images.
```

Comment:
Maybe put a note `(there are restrictions on image hosts, see below)`

Reply:
Where `see below` is a link to the heading below.

Reply:
I don't see in the client docs which nuget.exe/.NET CLI readme is supported in (I'm looking here: https://docs.microsoft.com/nuget/reference/msbuild-targets#packagereadmefile) . We do have this detail for icons (https://docs.microsoft.com/en-gb/nuget/reference/nuspec#icon). We should add it.